actions/websocket_actions.jsx

```diff
@@ -45,22 +45,27 @@ export function initialize() {
         return;
     }
 
-    let connUrl = getSiteURL();
+    let connUrl = '';
+    if (global.window.mm_config.WebsocketURL === '') {
```

Comment:
Can we remove the usage of the global variable here?

Reply:
Ya. Actually lets just wait for Jesse's PRs to be merged, then I can rebase ontop of that.

Reply:
@crspeller Those PRs are merged now

Reply:
The `actions/` subdirectory is part of Phase 2, which I'm tackling presently. If it's easier for everyone, I don't mind this change going in as-is and I can clean this file up holistically.

Reply:
Ok. If that's easier for you let's do that.